frontend/libs/studio-components/src/components/StudioTableLocalPagination/StudioTableLocalPagination.test.tsx

```diff
@@ -2,16 +2,20 @@ import React from 'react';
 import { render, screen, within } from '@testing-library/react';
 import userEvent from '@testing-library/user-event';
 import { StudioTableLocalPagination } from './StudioTableLocalPagination';
-import type { StudioTableLocalPaginationProps } from './StudioTableLocalPagination';
+import type { LocalPaginationProps } from './StudioTableLocalPagination';
 import { columns, rows } from '../StudioTableRemotePagination/mockData';
+import type { Rows, Columns } from '../StudioTableRemotePagination';
 
 describe('StudioTableLocalPagination', () => {
-  const paginationProps: StudioTableLocalPaginationProps['pagination'] = {
+  const paginationProps: LocalPaginationProps = {
     pageSizeOptions: [5, 10, 50],
-    pageSizeLabel: 'Rows per page',
-    nextButtonText: 'Next',
-    previousButtonText: 'Previous',
-    itemLabel: (num) => `Page ${num}`,
+    paginationTexts: {
+      pageSizeLabel: 'Rows per page:',
+      totalRowsText: 'Total number of rows:',
+      nextButtonAriaLabel: 'Next',
+      previousButtonAriaLabel: 'Previous',
+      numberButtonAriaLabel: (number) => `Page ${number}`,
+    },
```

Comment:
Maybe we could do something similar with the texts as done for StudioExpression? Then we can use variables all over the tests

Reply:
`paginationTexts` has been moved into `mockData.tsx` ✅